src/store/dapp-staking/actions.ts

```diff
@@ -798,15 +798,15 @@ const actions: ActionTree<State, StateInterface> = {
         commit('setMaxUnlockingChunks', maxUnlockingChunks?.toNumber());
 
         // Check if dapps staking is enabled.
-        let isPalletDisabled = false;
-        try {
-          const isDisabled = await $api.value.query.dappsStaking.palletDisabled<bool>();
-          isPalletDisabled = isDisabled.valueOf();
-        } catch {
-          // palletDisabled storage item is not supported by a node;
-        }
-
-        commit('setIsPalletDisabled', isPalletDisabled);
+        // let isPalletDisabled = false;
+        // try {
+        //   const isDisabled = await $api.value.query.dappsStaking.palletDisabled<bool>();
+        //   isPalletDisabled = isDisabled.valueOf();
+        // } catch {
+        //   // palletDisabled storage item is not supported by a node;
+        // }
+
+        commit('setIsPalletDisabled', true);
```

Comment:
Can you make it so that it can allow Shiden users to continue using dapps staking while only filter astar?

Reply:
@bobo-k2 
I see there is no `palletDisabled` query in Astar. How about:
```
// Memo: "Astar" or allocate node version like `isIndividualClaim`
if(network === Astar){
  return true
}else{
  const isDisabled = await $api.value.query.dappsStaking.palletDisabled<bool>();
  return isDisabled.valueOf();
}
```

Reply:
Sure, that works

Reply:
I made a quick fix. I will merge now and polish later

Reply:
Added some improvements